.builds/openbsd_0.yml

```diff
@@ -18,6 +18,12 @@ environment:
 tasks:
 - setup: |
     cd Nim
+    if [ "${GITHUB_PR_NUMBER+1}" ]; then
```

Comment:
see note above: `## do not edit directly; auto-generated by `nim r tools/ci_generate.nim`

you need to edit `tools/ci_generate.nim` instead

Reply:
I must have missed that comment, will update the script today and rebuild.

Not sure RE changing FreeBSD over to it - seems like it creates more work than it's worth really.